### ISSUE TITLE
Tag should be inside SELECT statement

### ISSUE THREAD

**Issue by jzabroski** [VariableNotFound/ef6-tagwith#6]
Hi,

Thanks for implementing this for EF6!

I was the one who originally proposed the EFCore TagWith API.  After it was released, we discovered that the SQL Server Query Store eliminates comments outside the query text, but keeps comments inside the query.

Thus:

```sql
-- Get top 10 older friends with country

SELECT TOP(@__p_0) [friend].[Name] AS [FriendName], [friend].[Age], 
                   [friend.Country].[Name] AS [CountryName]
FROM [Friends] AS [friend]
LEFT JOIN [Countries] AS [friend.Country] ON [friend].[CountryId] = [friend.Country].[Id]
ORDER BY [friend].[Age] DESC
```

Should be:

```sql
SELECT TOP(@__p_0) [friend].[Name] AS [FriendName], [friend].[Age], /* Get top 10 older friends with country */
                   [friend.Country].[Name] AS [CountryName]
FROM [Friends] AS [friend]
LEFT JOIN [Countries] AS [friend.Country] ON [friend].[CountryId] = [friend.Country].[Id]
ORDER BY [friend].[Age] DESC
```

or even 

```sql
SELECT /* Get top 10 older friends with country */
TOP(@__p_0) [friend].[Name] AS [FriendName], [friend].[Age],
                   [friend.Country].[Name] AS [CountryName]
FROM [Friends] AS [friend]
LEFT JOIN [Countries] AS [friend.Country] ON [friend].[CountryId] = [friend.Country].[Id]
ORDER BY [friend].[Age] DESC
```

**Comment by jmaguilar** [comment]
First of all, thanks for contributing and congrats for such great idea for the product :)

I didn't know that about the query store, so thanks for pointing it out. At a first sight, it doesn't seem to be too complex to implement in the component. 

I'll take a look at it and study one way to enable this "query store" mode, just for not breaking the current behavior for people that could be using it.

Thank you very much!

**Comment by jzabroski** [comment]
@jmaguilar Great! I also opened #7 .  If you do not have time to maintain, consider asking Jonathan Magnan at ZZZProjects.  He sells support for this kind of stuff, and he built EF6 temporal table support for us!  Highly recommended.

**Comment by jmaguilar** [comment]
Thank you. I'll try to work on the component next Christmas holidays, so I hope to release a new version soon :)

**Comment by jmaguilar** [comment]
Feature included in version 1.2.0. Thanks!
